README.md

```diff
@@ -3,6 +3,9 @@
 # Awesome Books
 
 > Awesome books is a simple project that displays new books when a user updates them.
+![screenshot](./images/spaadd.png)
+![screenshot](./images/spaList.png)
+![screenshot](./images/spacontact.png)
 
```

Comment:
- [ ] I recommend adding a `license `in the License section of the Readme file.

Reply:
The license in the README.md file was already there. Kindly check that.